project-base/app/webpack.config.js

```diff
@@ -82,6 +86,7 @@ config.resolve.alias = {
     'jquery': path.resolve(path.join(__dirname, 'node_modules', 'jquery')),
     'jquery-ui-styles': path.resolve(path.join(__dirname, 'node_modules', 'jquery-ui')),
     'bazinga-translator': path.resolve(path.join(__dirname, 'node_modules', 'bazinga-translator')),
-    'jquery-ui-nested-sortable': path.resolve(path.join(__dirname, 'node_modules', 'nestedSortable'))
+    'jquery-ui-nested-sortable': path.resolve(path.join(__dirname, 'node_modules', 'nestedSortable')),
+    'icons': path.resolve(path.join(__dirname, 'assets/icons/tabler'))
```

Comment:
Hm, you can import icons from different namespace (like bootstrap) and then you will not have those icons here. I would change path only to `assets/icons`

Reply:
changed in 3709373b86 and 2100dcd415